internal/file/file.go

```diff
@@ -258,23 +258,23 @@ func CopyFileWithPerm(ctx context.Context, src, dst string, perm fs.FileMode) (n
 }
 
 func WriteFile(ctx context.Context, target string, r io.Reader, perm fs.FileMode) (n int64, err error) {
-	return doWriteFile(ctx, target, r, os.O_CREATE|os.O_WRONLY|os.O_SYNC, perm)
+	return writeFileSync(ctx, target, r, os.O_CREATE|os.O_WRONLY|os.O_SYNC, perm)
 }
 
 func OverWriteFile(ctx context.Context, target string, r io.Reader, perm fs.FileMode) (n int64, err error) {
-	return doWriteFile(ctx, target, r, os.O_CREATE|os.O_TRUNC|os.O_WRONLY|os.O_SYNC, perm)
+	return writeFileSync(ctx, target, r, os.O_CREATE|os.O_TRUNC|os.O_WRONLY|os.O_SYNC, perm)
 }
 
 func AppendFile(ctx context.Context, target string, r io.Reader, perm fs.FileMode) (n int64, err error) {
-	return doWriteFile(ctx, target, r, os.O_CREATE|os.O_APPEND|os.O_RDWR|os.O_SYNC, perm)
+	return writeFileSync(ctx, target, r, os.O_CREATE|os.O_APPEND|os.O_RDWR|os.O_SYNC, perm)
 }
 
-func doWriteFile(ctx context.Context, target string, r io.Reader, flg int, perm fs.FileMode) (n int64, err error) {
+func writeFileSync(ctx context.Context, target string, r io.Reader, flg int, perm fs.FileMode) (n int64, err error) {
```

Comment:
```suggestion
func writeFile(ctx context.Context, target string, r io.Reader, flg int, perm fs.FileMode) (n int64, err error) {
```

Reply:
I renamed this function because of it is warned by deepsource which has `confusing name` function `WriteFile()` exists in this package. (line 260)
If I rename it to `writeFile()` the warning will occur again.
What do you think about it?

Reply:
I see. thank you for your explanation.
I will close this comment, thank you.